src/test/java/uk/nhs/hee/tis/trainee/sync/service/ReferenceSyncServiceTest.java

```diff
@@ -74,22 +76,12 @@ void shouldNotSyncRecordWhenTableNotSupported() {
     verifyNoInteractions(restTemplate);
   }
 
-  @Test
-  void shouldNotSyncRecordWhenOperationNotSupported() {
```

Comment:
Can you create a test like this in the `RecordServiceTest` and it's at least technically possible that `record.operation` is null.

Reply:
Done